main.py

```diff
@@ -1,10 +1,59 @@
+import json
+from typing import Union
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+        for player_nickname, player_data in data.items():
```

Comment:
I'm not if it's an issue, but I would exit the `with` block as soon as I finished the last operation with file. No need to keep useless references in memory, right?